barretenberg/cpp/src/barretenberg/vm/avm/trace/trace.cpp

```diff
@@ -3762,7 +3768,7 @@ std::vector<Row> AvmTraceBuilder::finalize()
      * ONLY FIXED TABLES FROM HERE ON
      **********************************************************************************************/
 
-    Row first_row = Row{ .main_sel_first = FF(1), .mem_lastAccess = FF(1) };
+    const Row first_row = Row{ .main_sel_first = FF(1), .mem_lastAccess = FF(1) };
```

Comment:
If we only use this once, better to make it non-const and do a std::move in the insert (or actually just do `{...}` in the insert and not have a variable for this.

Reply:
Yep I inlined it.